src/configure/helper/commonHelper.ts

```diff
@@ -17,7 +30,7 @@ export function generateDevOpsProjectName(repositoryName?: string): string {
     // project name cannot end with . or _
     suffix = suffix.replace(/\.[\.]*$/, '').replace(/^_[_]*$/, '');
 
-    let projectName = `AzurePipelinesProject.${suffix}`;
+    let projectName = `AzurePipelines-${suffix}`;
```

Comment:
should we not have random suffix for project name?

Reply:
Since this is a new org/new project scenario, there would not be any name clash for project as the organization is newly created. Therefore I have not added unique suffix.